db/Pos/DB/Class.hs

```diff
@@ -80,7 +80,7 @@ import           Control.Monad.Trans.Lift.Local (LiftLocal (..))
 import           Control.Monad.Trans.Resource   (ResourceT)
 import           Data.Conduit                   (Source)
 import qualified Database.RocksDB               as Rocks
-import qualified Ether
+import           Ether.Internal                 (HasLens (..))
```

Comment:
Initially I planned on removing `ether` completely, but it's just too much work to do in one take.